daemon/src/projection.rs

```diff
@@ -316,6 +321,7 @@ impl Cfd {
         match event.event {
             ContractSetupStarted => {
                 self.state = CfdState::ContractSetup;
+                self.creation_timestamp = event.timestamp;
```

Comment:
I guess this is ok for now, alternatively, we could get the timestamp of the first event. Would need to check the version of the aggregate for that.

Reply:
Wouldn't this event be the first event (that counts) all the time. `OfferRejected` is the only other event that could come before this but It won't show in the tiles.

Reply:
> Wouldn't this event be the first event (that counts) all the time. `OfferRejected` is the only other event that could come before this but It won't show in the tiles.

Yep, that is correct. It is a question of how to best express the intent.

If we have a bit of code that does:

```rust
if self.version == 0 {
	self.creation_timestamp = event.timestamp;
}
```

Then this expresses more clearly that we are defining the concept of the creation timestamp as the timestamp of the first event.
If we tie the logic to the specific event `ContractSetupStarted`, then the reader has to know that this is the first event to understand why that logic is there and not somewhere else.

I don't mind much either way :)

Reply:
I think it is more resilient to express this with `self.version == 0` and not use `ContractSetupStarted`.

Reply:
I think I agree. @rishflab do you mind doing this in a follow-up? Also I think we now can/should remove the "ORDER BY" statement here: https://github.com/itchysats/itchysats/blob/78cae83c66ff2b7f9f7cb8f10019c7baaec2e04c/daemon/src/db.rs#L274